src/state-summary/state-card-lock.html

```diff
@@ -0,0 +1,42 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<link rel="import" href="../components/entity/state-info.html">
+<link rel="import" href="../components/entity/ha-entity-lock.html">
+
+<dom-module id="state-card-lock">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
```

Comment:
is="custom-style" is old syntax and can be removed

Reply:
This was just a clone from state-card-toggle. I removed it.